spk/syncthing/Makefile

```diff
@@ -8,11 +8,11 @@ DEPENDS = cross/$(SPK_NAME)
 
 UNSUPPORTED_ARCHS = $(PPC_ARCHS)
 
-MAINTAINER = SynoCommunity
+MAINTAINER = acolomb
 DESCRIPTION = Automatically sync files via secure, distributed technology.
 DESCRIPTION_FRE = Synchronisation automatique de fichiers via une technologie sécurisée et distribuée.
 DISPLAY_NAME = Syncthing
-CHANGELOG = "Update syncthing to v1.18.5.  Add wizard page to preconfigure credentials for the Web GUI."
+CHANGELOG = "Update syncthing to v1.18.5.  Add wizard page to preconfigure credentials for the Web GUI.  Integrate with the certificate management UI in Control Panel."
```

Comment:
I beleive the package version should be bumped: https://github.com/SynoCommunity/spksrc/pull/4930/files#diff-27e7feeb4837853b261e74b395eb80e9e11bc0b1ab5a4f9425985437f4e515b2R3

Reply:
The revision was already bumped in #4995, but the package has not been published since then.  So I think we'll just keep piling up new features under the same `SPK_REV` until they are actually made available?

What's up with the link?  It only takes me to the diff view?

Reply:
> What's up with the link? It only takes me to the diff view?

It seems because I pointed to hidden line. My bad. https://github.com/SynoCommunity/spksrc/blob/b92382700ea70d436740523865e3b434e5aa9bad/spk/syncthing/Makefile#L3